inference/core/models/roboflow.py

```diff
@@ -281,7 +282,7 @@ def model_artifact_bucket(self):
         return INFER_BUCKET
 
     def download_model_artifacts_from_roboflow_api(self) -> None:
-        logger.debug("Downloading model artifacts from Roboflow API")
+        logger.debug("Downloading model artifacts from Roboflow API %s:", self.endpoint)
```

Comment:
Does self.endpoint contain the auth token for GCS?

Reply:
wierd name but its really just modelid.  like the `model/version` string or similar thing for versionless models.  its `model_id` after `resolve_roboflow_model_alias` has happened essentially.  we log it all over the place